tests/task/test_calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,227 @@
+                testPipelineConfigDir, "testCalcZernikesCwfsSetupPipeline.yaml"
+            )
+
+            pipeCmd = writePipetaskCmd(
+                cls.repoDir,
+                cls.runName,
+                instrument,
+                collections,
+                pipelineYaml=pipelineYaml,
+            )
+            # Use the same detector selection as CWFS tests
+            pipeCmd += ' -d "detector IN (191, 192)"'
+            runProgram(pipeCmd)
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """
+        Clean up test data after all tests are complete.
+        """
+        if cls.runName == "run1":
+            cleanUpCmd = writeCleanUpRepoCmd(cls.repoDir, cls.runName)
+            runProgram(cleanUpCmd)
+
+    def setUp(self) -> None:
+        """
+        Set up individual test environment.
+        
+        This method configures the task with your specific estimation algorithm
+        and loads test data for each test method.
+        """
+        self.repoDir = '/home/peterma/research/Rubin_AO_ML/training/butler'
+        self.config = CalcZernikesTaskConfig()
+        
+        # TODO: Replace with your specific estimation task
+        # self.config.estimateZernikes.retarget(YourEstimationTask)
+        self.config = CalcZernikesNeuralTaskConfig()
+        self.config.wavenet_path = '/home/peterma/research/Rubin_AO_ML/training/finetune_logs/lightning_logs/version_0/checkpoints/best_finetuned_wavennet.ckpt'
+        self.config.alignet_path = '/home/peterma/research/Rubin_AO_ML/training/alignnet_logs/lightning_logs/version_0/checkpoints/best_alignnet_120.ckpt'
+        self.config.aggregatornet_path = '/home/peterma/research/Rubin_AO_ML/training/aggregator_logs/lightning_logs/version_0/checkpoints/best_aggregator.ckpt'
+        self.config.dataset_param_path = '/home/peterma/research/Rubin_AO_ML/TARTS/TARTS/dataset_params.yaml'
+        self.config.device = 'cpu'
+
+        # Initialize the neural task
+        self.task = CalcZernikesNeuralTask(config=self.config, name="Neural Task")
+        
+        # Set the run name for data collection
+        self.runName = "pretest_run_cwfs"
+        
+        # Initialize Butler and registry for data loading
+        self.butler = Butler.from_config(self.repoDir)
+        self.registry = self.butler.registry
+
+        # Define data IDs for extra and intra focal exposures
+        # Using adjacent detectors (191 and 192) from the same visit
+        self.visitNum = 2031063000001
+        self.dataIdExtra = {
+            "instrument": "LSSTCam",
+            "detector": 191,
+            "exposure": self.visitNum,
+        }
+        self.dataIdIntra = {
+            "instrument": "LSSTCam",
+            "detector": 192,
+            "exposure": self.visitNum,
+        }
+
+    def test_load_model(self) -> None:
+        """
+        Test we can load the models
+        """
+        print(self.task)
+
+    def test_load_intra_extra_exposures(self) -> None:
```

Comment:
Please, beware in the entire PR, ts_wep uses camelCase, so we need to be consistent with that. In this particular case, the name should be testLoadIntraExtraExposures